chapters/05-cli/5j-v2.md

```diff
@@ -0,0 +1,3 @@
+# Descriçao #
+
+O comando *v2* ira imprimir informações relevante para iniciar uma aplicação na versão V2.
```

Comment:
Opa, desculpe pela demora...
Irei corrigir ainda hoje e já faço o commit.

Em 20 de jul de 2016 11:22 AM, "Gabriel Lima" <notifications@github.com>
escreveu:

> In chapters/05-cli/5j-v2.md
> <https://github.com/IonicBrazil/ionic2-docs/pull/53#discussion_r71533976>:
>
> > @@ -0,0 +1,3 @@
> > +# Descriçao #
> > +
> > +O comando *v2* ira imprimir informações relevante para iniciar uma aplicação na versão V2.
>
> *irá*
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/IonicBrazil/ionic2-docs/pull/53/files/e3a94624f79c06d2862c09feb8ccf4b85969226a#r71533976>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABR4t89MlmCcAOvlTQw4M9xwBXC2GyJ2ks5qXi8_gaJpZM4IWwaL>
> .
>